opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -174,6 +174,21 @@ def add(self, value: metrics_api.ValueT, labels: Dict[str, str]) -> None:
     UPDATE_FUNCTION = add
 
 
+class UpDownCounter(Metric, metrics_api.UpDownCounter):
```

Comment:
With `UpDownCounter` now accepting any values, `Counter` needs some validations on update for negative values (similar to `SumObserver`)